File: Pemro1_Modul1/Modul1_Pyton/PRAK104-2510817320010-PutriDeissytaHerliyana.py
a = 400000
b = 350000

diskona = 13 / 100
diskonb = 21 / 100

hargaakhira = a - (a * diskona)
hargaakhirb = b - (b * diskonb)

print("Harga sepatu a adalah", a)
print("Harga sepatu b adalah", b)
print("Sepatu A mendapatkan diskon 13% sehingga harganya menjadi", int(hargaakhira))
print("Sepatu B mendapatkan diskon 21% sehingga harganya menjadi", int(hargaakhirb))

File: Pemro1_Modul1/Modul1_C/PRAK104-2510817320010-PutriDeissytaHerliyana.c
#include <stdio.h>

int main(){
    int A = 400000;
    int B = 350000;
    int hasil1 = A - (A*13/100);
    int hasil2 = B - (B*21/100);
    printf ("Harga sepatu A adalah %d\n", A);
    printf ("Harga sepatu B adalah %d\n", B);
    printf ("Sepatu A mendapat diskon 13%% sehingga harganya menjadi %d\n", hasil1);
    printf ("Sepatu B mendapat diskon 21%% sehingga harganya menjadi %d", hasil2);
return 0;
}